charms/knative-operator/src/manifests/auth_manifests.yaml.j2

```diff
@@ -676,6 +673,31 @@ rules:
       - deployments
     verbs:
       - deletecollection
+  # Eventing TLS
+  - apiGroups:
+      - "cert-manager.io"
+    resources:
+      - certificates
+      - issuers
+      - clusterissuers
+    verbs:
+      - create
+      - delete
+      - update
+      - list
+      - get
+      - watch
+  - apiGroups:
+      - "trust.cert-manager.io"
+    resources:
+      - bundles
+    verbs:
+      - create
+      - delete
+      - update
+      - list
+      - get
+      - watch
```

Comment:
We do not use `cert-manager` at all, so we could skip these. Also, in the near future we will remove `Roles, ClusterRoles` and bindings since all of our charms are deployed with `--trust` and this will not be needed anymore. For now we can leave these untouched. 

Note that we still want to keep the aggregated roles because those are propagated to the user namespaces by the `kubeflow-profile` controller so users can manipulate different resources.